client/src/App.js

```diff
@@ -0,0 +1,13 @@
+import { Fragment } from 'react';
+
+import './App.css';
+
+function App() {
+  return (
+    <Fragment>
+      <h1>Hello World</h1>
+    </Fragment>
```

Comment:
Technically you don't need to return this in a fragment since only one element is present anyways. https://reactjs.org/docs/fragments.html

Reply:
I was anticipating this fragment to contain multiple components (eg. pages, nav) in the future.